test/ccall.jl

```diff
@@ -123,7 +126,11 @@ function test_struct1{Struct}(::Type{Struct})
     b = Float32(123.456)
 
     a2 = copy(a)
-    x = ccall((:test_1, libccalltest), Struct, (Struct, Float32), a2, b)
+    if Struct === Struct1
```

Comment:
In practice, it's not an issue. When I ran PkgEval, and there weren't any packages that failed due to needing this.